src/scripts/components/embed/Embed.jsx

```diff
@@ -200,6 +223,7 @@ class Embed extends Component {
                     options={waveSurferOptions}
                     playing={this.state.playing}
                     pos={pos}
+                    {...wavesurferProps}
```

Comment:
whoa, cool shorthand! I didn't know about this one.

Reply:
Yeah! I just learned about it, it's pretty neat :)